cltk/text_reuse/levenshtein.py

```diff
@@ -133,11 +134,10 @@ def ratio(string_a, string_b):
             from fuzzywuzzy import fuzz
 
         except ImportError as imp_err:  # pragma: no cover
-            message = "'fuzzywuzzy' library required for this module: %s. Install with `pip install fuzzywuzzy python-Levenshtein`" % imp_err
+            message = "'fuzzywuzzy' library required for this module: %s. Install with " \
```

Comment:
OK @clemsciences  I see that you're not replacing anything using the `.ratio()` method. I am OK with all this.

cc @todd-cook

Reply:
This function was only implemented by 'fuzzywuzzy' and as I only wanted to re-use already existing internal functions, then I have not modified the `ratio` method.

Reply:
> This function was only implemented by 'fuzzywuzzy' and as I only wanted to re-use already existing internal functions, then I have not modified the ratio method.

Their api is non-transparent. fuzzywuzzy does not include python-Levenshtein in its requirements b/c some systems don't handle C code well; instead fuzzywuzzy checks if python-Levenshtein is available and calls that silently, otherwise it silently uses its own native python. See on their github: "The Levenshtein Python C extension module contains functions for fast computation of Levenshtein distance and string similarity".

Since we expect our users to want to compare longer strings, we ought to encourage them to install this python-Levenshtein, too.

Reply:
This pull request just proposes to avoid python-Levenshtein for functions that were already coded in CLTK.

Reply:
Imo, this is a good change. We don't  care much about speed and introducing an optional dependency for relatively core functions doesn't make much sense. Alternatively, we could improve the docs a little bit to make them aware of the dependency